nautobot/dcim/views.py

```diff
@@ -2432,7 +2432,7 @@ class CableBulkDeleteView(generic.BulkDeleteView):
 
 
 class ConsoleConnectionsListView(generic.ObjectListView):
-    queryset = ConsolePort.objects.filter(_path__isnull=False).order_by("device")
+    queryset = ConsolePort.objects.filter(_path__isnull=False).order_by("device__identifier", "name")
```

Comment:
Two things:

1. How is this even working? `Device.identifier` isn't a concrete database field (it's a property on the `Device` model
2. Doing ordering on nested model fields results in a table join. Assuming this even works, just wanted to be mindful of introducing this for these views.

Reply:
1) I have no idea -- it seems to work, or at least doesn't complain at me? I'm open to alternate suggestions though - ordering by `device` (PK?) is obviously kinda non-deterministic so I wanted something better. I suppose I could just order by `name` if you'd prefer?